lib/app/modules/home/views/home_view.dart

```diff
@@ -608,3 +610,71 @@ class HomeView extends GetView<HomeController> {
     );
   }
 }
+
+
+
+// ...
+
+Future<void> checkOverlayPermissionAndNavigate(BuildContext context) async {
+  if (!(await FlutterForegroundTask.canDrawOverlays) || !(await Permission.notification.isGranted)) {
+    showDialog(
```

Comment:
Since we've been using GetX for dialogs, I think we should keep it consistent unless there's any significant reason to switch :P